localstack/services/awslambda/lambda_executors.py

```diff
@@ -638,6 +647,18 @@ def execute_java_lambda(self, event, context, handler, main_file):
         return result, log_output
 
 
+class Util:
+
+    @staticmethod
+    def get_java_opts(port):
+        opts = config.LAMBDA_JAVA_OPTS
+        if opts.find('address'):
+            java_opts = opts.replace('address=', ('address=%s' % port))
```

Comment:
One more suggestion here. How about we provide a placeholder like `_debug_port_` that the user can use in their `LAMBDA_JAVA_OPTS` configuration?

This would make the replacement logic here a bit more intuitive/explicit, instead of having the user specify a prefix like `address=`

Also, we could then check for the existence of this placeholder, and only allocate a container port if the placeholder is actually contained in the java opts.

Let me know if you can still make this change - otherwise I can add it in a separate commit.

Reply:
Ok, do you mean provide placeholder in LAMDA_JAVA_OPTS docker compose variable? And if exist, replace for address and port, right?